public/css/style2.css

```diff
@@ -0,0 +1,210 @@
+/*--
```

Comment:
I don't like this naming convention. `style.css` and `style2.css` are not very informative names of the files.